src/cn-toast.js

```diff
@@ -0,0 +1,56 @@
+(function() {
+  'use strict';
+
+  angular.module('cn.ui')
+      .controller('toastController', [
+        '$rootScope', 'toaster',
```

Comment:
We should be using `'ngInject'` or `Controller.$inject = []` style syntax

Also since ng 1.5, convention is to use components rather than directives